src/packagers/SourceMapPackager.js

```diff
@@ -17,7 +17,10 @@ class SourceMapPackager extends Packager {
   async end() {
     let file = path.basename(this.bundle.name);
     await this.dest.write(this.sourceMap.stringify(file));
-    await super.end();
+    this.stopend();
```

Comment:
what about just doing `return super.end();` here?

Reply:
when the babel transform .the await super change to function *(){ super.....}
so make a function .do super job. and then use this to called that function.
just a one of the way to edit them.have many ways to do that....
i think if we want to make it support node version <8.
we should do better and think more things~~~
Otherwise it does not support version <8 just fine